server/database/commands.go

```diff
@@ -0,0 +1,64 @@
+package database
+
+import (
+	"database/sql"
+
+	"github.com/rs/zerolog/log"
+)
+
+func InsertMoveCommand(db *sql.DB, id int64, commandType string, target int64) error {
+	log.Info().Msgf("insert into t_command (ship, type, target) values (%d, %s, %d)", id, commandType, target)
+	_, err := db.Exec(`insert into t_command (ship, type, target) values (?, ?, ?)`, id, commandType, target)
```

Comment:
use `replace` instead of insert
https://dev.mysql.com/doc/refman/8.0/en/replace.html